keyboards/boardwalk/boardwalk.h

```diff
@@ -0,0 +1,82 @@
+#ifndef BOARDWALK_H
+#define BOARDWALK_H
+
+#include "quantum.h"
+
+#define LAYOUT( \
```

Comment:
Could you rename this to be represent the actualy layout type? 

Eg:

```suggestion
#define LAYOUT_ortho_5x14( \
```

Reply:
Updated in https://github.com/qmk/qmk_firmware/pull/5195/commits/e23d9758c802d43be8763bd7982cfec5220f1d82